### DISCUSSION TITLE
Problem setting cell borders

### DISCUSSION THREAD

**Discussion by jzieba01** [PHPOffice/PhpSpreadsheet#1879] [Q&A] [answered]
I am having a problem trying to apply borders to cells.  I get the following error message:

"Fatal error: Uncaught Error: Class 'Border' not found in ..."

`$styleArray = array(
    'borders' => array(
        'allborders' => array(
            'borderStyle' => Border::BORDER_MEDIUM,
            'color' => array('argb' => '000000'),
        ),
    ),
);
$activeSheet->getStyle('A2:J15')->applyFromArray($styleArray);
`
Any idea on how to fix this?  This is pretty much straight from the docs.

**Comment by MarkBaker** [comment, marked as answer]
The `Border` class is namespaced in `PhpOffice\PhpSpreadsheet\Style`.

Do you have a `use PhpOffice\PhpSpreadsheet\Style\Border;` at the top of your script? Otherwise use a FQDN.
